crates/jstz_cli/src/jstz.rs

```diff
@@ -140,7 +141,22 @@ impl JstzClient {
             }
 
             // tokio sleep
-            sleep(Duration::from_millis(100)).await;
+            sleep(Duration::from_millis(200)).await;
```

Comment:
What motivated the change to 200?

Reply:
I noticed that at 100ms we were spamming the jstz-node, happy to change it back